frontend/src/components/SignUp/SignUp.js

```diff
@@ -16,41 +22,52 @@ class SignUp extends Component {
     }));
   }
 
+  handleChange = (event) => {
+    this.setState({ [event.target.name] : event.target.value});
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault()
+    createUser(this.state).then(response => {
+      console.log(response)}).catch(error => {console.log(error.response.data)})
+  }
+
+
   render() {
     return (
       <div className="leftLoginCard loginCard">
         <h3 className="loginHeader">Create an Account</h3>
-        <form className="form-signInUp">
+        <form className="form-signInUp" onSubmit={this.props.mimicLogin}>
           <div className="form-group row">
               <div className="name col">
                 <div className="name row">
-                  <input type="text" id="inputFirstName" className="sign-in-input firstName col-md form-control-lg" required="" autoFocus="" placeholder="First" onFocus={(event) => (event.target.setAttribute("placeholder", ""))}
-onBlur={(event) => (event.target.setAttribute("placeholder", "First"))}/>
+                  <input type="text" id="inputFirstName" name="firstName" className="sign-in-input firstName col-md form-control-lg" placeholder="First" onFocus={(event) => (event.target.setAttribute("placeholder", ""))}
+onBlur={(event) => (event.target.setAttribute("placeholder", "First"))} onChange={(event) => this.handleChange(event)}/>
```

Comment:
Same thing from the previous PR. I think `onChange=this.handleChange` would work, and it'd be stellar to refactor this repeated pattern into a function or small component.

Reply:
For sure. I'll fix the function onChange now, but I again, I think that should be a separate ticket.

Reply:
And done

Reply:
We can make it a separate ticket if you think this would block other work. Were you thinking of doing that this sprint, as well?

I do think if the intent is to come back to this down the road, we should acknowledge there's a good chance that's not going to happen. The calculus changes once it becomes technical debt instead of feature work. But I understand the desire to finish something and move on.

Reply:
Lemme see what I can do here . . . I'm going to try and refactor this into a little component. That makes the most sense to me for some reason